README.md

```diff
@@ -199,6 +199,14 @@ To use `stdlib` within your `fpm` project, add the following lines to your `fpm.
 stdlib = { git="https://github.com/fortran-lang/stdlib", branch="stdlib-fpm" }
 ```
 
+> **Warning**
+> 
+> Fpm 0.8.2 and later implements stdlib as a *metapackage*.
```

Comment:
Please use `0.9.0`, it’s the release metapackages went officially into

Reply:
We have user documentation at https://fpm.fortran-lang.org/en/spec/metapackages.html 

can I ask you to use that link? I think it’s better than referring to a developer page

Reply:
Would it be better to have the option in from to use stdlib from a branch as advertised here and have it as metapackage as well. This way we don't break backwards compatibility.

Reply:
https://github.com/fortran-lang/fpm/issues/926